### PR TITLE
Add healer script

### PR DESCRIPTION
**Pull request by jandersson**
Based on fest-healer. Modified to use healme, common-healing and
common-arcana to offload lots of the logic. Additions include the
pausing and starting of scripts when healing, meaning one could use this
while running crossing-training.

### CODE REVIEW

File: healer.lic [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    echo "Healing #{name}" if debug?
+    @patient = name
+    return if @touched && @touched >= Time.now
+
+    unless @touched
+      @touched = Time.now + 12
+      pause 1
+      waitrt?
+      fput("touch #{name}")
+      put("transfer #{name} vit quick")
```
Comment (bmcneff) [inline healer.lic]: fput, not put

Comment (jandersson) [reply healer.lic]: This is from the original script. My understanding is that put is when we dont care about a response from the game. As someone using the script for several hours, this was never an issue.

Comment (bmcneff) [reply healer.lic]: 👍

Comment (jandersson) [reply healer.lic]: Appreciate the review nonetheless :) I think the argument on `put` for vitality is, 'we'll get it if we get it, its time to move on to wounds'

File: healer.lic [hunk 2]
```diff
@@ -0,0 +1,139 @@
+=begin
+  Documentation:
```
Comment (rpherbig) [inline healer.lic]: Will this be documented on elanthipedia?

File: healer.lic [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+=end
+no_pause_all
+custom_require.call %w[events common common-arcana common-healing common-items]
```
Comment (rpherbig) [inline healer.lic]: Why is `events` required?

File: healer.lic [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+class Healer
+  include DRC
+  include DRCH
+  include DRCI
+  include DRCA
+
+  def empty_hands
```
Comment (rpherbig) [inline healer.lic]: What do you think about using `stow_hands` from DRCI instead?

File: healer.lic [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    echo 'Emptying hands' if debug?
+    @left = checkleft
+    echo "@left: #{@left}" if debug?
+    @right = checkright
+    echo "@right: #{@right}" if debug?
+    fput('stow left') if checkleft
```
Comment (rpherbig) [inline healer.lic]: What do you think about using `stow_hands` from DRCI here?

**Pull request merged**